examples/c/src/main.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+#include "sum.h"
+
+int main(int argc, char *argv[]) {
+	int n1 = 100;
+    int n2 = 200;
```

Comment:
It looks like there's a mix between tabs and spaces in this file; can you change this to consistently use one or the other.

Reply:
Okay..I will fix them.